src/pages/UserProfile/SelfProfile/SelfProfilePage.tsx

```diff
@@ -24,6 +24,17 @@ export default function SelfProfilePage(): JSX.Element {
   const userProfileImage = useRecoilValue(userProfileImageAtom);
   const userCoverImage = useRecoilValue(userCoverImageAtom);
 
+  const { disconnect, connected } = useWallet();
+
+  async function disconnectAll() {
+    try {
+      await SignoutWithWallet({ disconnect, connected });
+      /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
+    } catch (error: any) {
```

Comment:
Im pretty sure you dont need to declare the type. I think you can simply to:

} catch (error) {

And it will work. That allows you to remove the /*eslint-disable-blah...*/

Reply:
whot?